frontend/src/components/Chat.tsx

```diff
@@ -60,6 +61,9 @@ const Chat = () => {
   const [questions, setQuestions] = useState<Question[]>([]);
 
   const [quizMessage, setQuizMessage] = useState('');
+  const [reactions, setReactions] = useState({} as { [reactionId: number]: Reaction });
```

Comment:
proponuję wprowadzenie czegoś w rodzaju:
```suggestion
  interface ReactionObject {
    [reactionId: number]: Reaction;
  }
  const [reactions, setReactions] = useState({} as ReactionObject);
```

Reply:
ok, zmieniłem